nx584/controller.py

```diff
@@ -346,6 +346,13 @@ def _get_user(self, number):
         return self.users[number]
 
     def process_msg_3(self, frame):
+        try:
+            zone_name_update = self._config.getboolean('config', 'zone_name_update')
+        except configparser.NoOptionError:
+            zone_name_update = True
+            self._config.set('config', 'zone_name_update', zone_name_update)
+        if not zone_name_update:
+            return
```

Comment:
By returning here, you skip logging the fact that we even received this frame from the panel. That could get very confusing during debugging if someone doesn't realize that having this flag set means we will completely silently ignore these frames. The better thing would be to just avoid running L360 (and L362) if the flag is set, but still log that we received it.